content/en/docs/ops/best-practices/security/index.md

```diff
@@ -129,13 +177,26 @@ apiVersion: v1
       ...
 {{< /text >}}
 
-### Less common normalization configurations
+### Mitigation for unsupported normalization
+
+This section describes various mitigations for unsupported normalization. These could be useful when you find a specific
+normalization that is not supported by Istio.
+
+Please make sure you understand the mitigation thoroughly and use it carefully. Some mitigation is just best-effort workaround
```

Comment:
I'm not sure I like the second sentence and it seems we are documenting untested, not our best effort items. It also needs to be reworked grammatically.

Reply:
Updated the wording, I just want to give the user a heads up that some mitigations are out the scope of Istio.